zrml/global-disputes/Cargo.toml

```diff
@@ -15,7 +16,7 @@ num-traits = { workspace = true, optional = true }
 
 [dev-dependencies]
 env_logger = { workspace = true }
-pallet-balances = { workspace = true, features = ["default"] }
+pallet-balances = { workspace = true, features = ["default", "insecure_zero_ed"] }
```

Comment:
Do we really want that?

https://github.com/paritytech/polkadot-sdk/blob/c8d2251cafadc108ba2f1f8a3208dc547ff38901/substrate/frame/balances/src/lib.rs#L274-L279

Reply:
It should never be done in prod but is fine for tests. I used the following approach: If tests within a pallet used zero ED before (i.e. `type ExistentialDeposit = ()` in `pallet_balances`), then I left it like that and enabled zero ed for tests cases. If t was non zero ED before, then I left it and no further action was necessary.